.github/workflows/release-bin.yaml

```diff
@@ -236,16 +363,34 @@ jobs:
         if: matrix.os == 'ubuntu-20.04'
         uses: svenstaro/upload-release-action@v2
         with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          repo_token: ${{ secrets.ACTION_TOKEN }}
```

Comment:
@xyephy sorry a few more questions:

did you change `GITHUB_TOKEN` FOR `ACTION_TOKEN` for some specific reason?

I'm trying to run a mock release on my fork and got [this error](https://github.com/plebhash/stratum/actions/runs/8806935564/job/24172831630):
```
Run svenstaro/upload-release-action@v2
Error: Input required and not supplied: repo_token
```

I'm not necessarily asking you to revert this change yet, just trying to understand the motivation behind it.
Also because if we merged it like this, we would need to change the token in the repository settings.

Reply:
> @xyephy sorry a few more questions:
> 
> did you change `GITHUB_TOKEN` FOR `ACTION_TOKEN` for some specific reason?
> 
> I'm trying to run a mock release on my fork and got [this error](https://github.com/plebhash/stratum/actions/runs/8806935564/job/24172831630):
> 
> ```
> Run svenstaro/upload-release-action@v2
> Error: Input required and not supplied: repo_token
> ```
> 
> I'm not necessarily asking you to revert this change yet, just trying to understand the motivation behind it. Also because if we merged it like this, we would need to change the token in the repository settings.

had changed cause of my environment, let me revert to GITHUB_TOKEN

Reply:
@plebhash  you can now test